### PR TITLE
Fix the example app

### PR DESCRIPTION
**Pull request by kevinrenskers**
- Set minimum iOS version to 13, same as the framework itself, fixes the error `Compiling for iOS 12.0, but module 'UINotifications' has a minimum deployment target of iOS 13.0`
- Force to light mode, as the UI breaks when the device it set to Dark Mode
- Fix compiler error because of incomplete UINotificationStyle comformance

### CODE REVIEW

**Comment by kevinrenskers** [comment]
The example app now compiles and runs again which is a great improvement, but showing a notification crashes the app:

`UINotifications-Example[21937:8094130] failure in dealloc of <BSServiceConnectionEndpointMonitor:0x6000017f51d0> (BSServiceConnectionEndpointMonitor.m:58) : must call invalidate before dealloc`

**Comment by BasThomas** [comment]
Haven't been able to pinpoint the issue, but did find https://github.com/nstack-io/nstack-ios-sdk/issues/83.

**Comment by BasThomas** [comment]
It seems to crash before the "ease in out presenter"'s animation calls its `completion` block, but even when completely disabling the code in its `present()` function, we can still observe this crash.

**Comment by BasThomas** [comment]
It seems like it is something in the internal `handle(_:)` function that causes this crash.

**Comment by BasThomas** [comment]
Particularly, creating the `presentationContext`.

**Comment by BasThomas** [comment]
This line crashes. https://github.com/WeTransfer/UINotifications/blob/96e571ccada2dafe9dc202665687e5a5eeb62897/Sources/UINotificationPresentationContext.swift#L78

**Comment by BasThomas** [comment]
Which leads to the crash in here https://github.com/WeTransfer/UINotifications/blob/96e571ccada2dafe9dc202665687e5a5eeb62897/Sources/UINotificationCenter.swift#L98

**Comment by BasThomas** [comment]
Seems like it doesn't like us doing this, which I must say is not unexpected https://github.com/WeTransfer/UINotifications/blob/96e571ccada2dafe9dc202665687e5a5eeb62897/Sources/UINotificationCenter.swift#L92-L93

**Comment by BasThomas** [comment]
A fix would be to use `UIApplication.shared` directly; not sure how to best "silence" / do nothing in an application extension?

**Review by BasThomas — APPROVED** [review]

File: Example/UINotifications-Example/ViewController.swift [hunk 1]
```diff
@@ -47,6 +47,10 @@ enum NotificationStyle: UINotificationStyle {
             return #colorLiteral(red: 1, green: 0.431372549, blue: 0.431372549, alpha: 1)
         }
     }
+
+    var thumbnailSize: CGSize {
+        return .init(width: 100, height: 100)
+    }
```
Comment (wetransferplatform) [inline Example/UINotifications-Example/ViewController.swift:54]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

File: Example/UINotifications-Example/ViewController.swift [hunk 2]
```diff
@@ -34,11 +34,11 @@ enum NotificationStyle: UINotificationStyle {
             return .white
         }
     }
-    
+
```
Comment (wetransferplatform) [inline Example/UINotifications-Example/ViewController.swift:37]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

File: Example/UINotifications-Example/ViewController.swift [hunk 3]
```diff
@@ -78,13 +82,13 @@ final class ViewController: UIViewController {
...
     @IBOutlet private weak var automaticallyDismissSwitch: UISwitch!
     @IBOutlet private weak var addActionSwitch: UISwitch!
     @IBOutlet private weak var addButtonSwitch: UISwitch!
     @IBOutlet private weak var showsImageSwitch: UISwitch!
     @IBOutlet private weak var successStyleSwitch: UISwitch!
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
```
Comment (wetransferplatform) [inline Example/UINotifications-Example/ViewController.swift:94]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

**Pull request merged**